src/components/IndustryEquivalency.tsx

```diff
@@ -161,7 +160,7 @@ const IndustryEquivalency = () => {
     // GET EQUIVALENCY ARRAY
     /* ---------------------------------------------------------------- */
     const aquireSkillSet = (() => {
-        axios.get(back_end_url + '/equiv/portfolios/all/' + portfolio.id)
+        axios.get(url + '/equiv/portfolios/all/' + portfolio.id)
```

Comment:
@ddinh please store `url + "/equiv"` to a variable and pass here to minimize code duplication throughout this file